web/static/js/main.js

```diff
@@ -125,13 +116,6 @@ function get_top_ten_element_in_event(event_type, element, html_element_id) {
             window.myPolarArea = Chart.PolarArea(ctx, top_ten_graph_config);
             const downloadButton = ctx.nextSibling.nextSibling;
             downloadButton.hidden = false;
-},
-        error: function (jqXHR, textStatus, errorThrown) {
```

Comment:
error function needs to be there,maybe log or something to understand what went wrong

Reply:
ok will add it